core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -21,6 +21,9 @@
 
 class ContaoLoginFactory extends AbstractFactory
 {
+    public const TRUSTED_DEVICES_TOKEN_LIFETIME = 5184000;
+    public const TRUSTED_DEVICES_TOKEN_ID_PREFIX = 'contao_2fa_trusted_device';
```

Comment:
Is there a reason to override the default values? We could just use the default `scheb_two_factor.trusted_token_storage` and not override the cookie response listener parameters respectively.

Reply:
Fixed in c25176a8f.